connector-runtime/connector-runtime-core/src/main/java/io/camunda/connector/runtime/core/inbound/InboundConnectorReportingContext.java

```diff
@@ -21,9 +21,12 @@
 import io.camunda.connector.api.inbound.InboundConnectorContext;
 import java.util.List;
 import java.util.Queue;
+import java.util.function.Consumer;
 
 public interface InboundConnectorReportingContext extends InboundConnectorContext {
 
+  void loadCancellationCallback(Consumer<Throwable> cancellationCallback);
```

Comment:
I don't really get it. The consumer seems to always be the same (coming from `cancel` in the ActivatedExecutableManager). Did you introduce this to avoid a circular dependency?